test/functional/pdo_sqlsrv/pdo_ae_insert_pdoparam_money.phpt

```diff
@@ -3,84 +3,68 @@ Test for inserting and retrieving encrypted data of money types
 --DESCRIPTION--
 Use PDOstatement::bindParam with all PDO::PARAM_ types
 --SKIPIF--
-<?php require('skipif_versions_old.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-include 'MsCommon.inc';
-include 'AEData.inc';
-
+require_once("MsCommon_mid-refactor.inc");
+require_once("AEData.inc");
 $dataTypes = array( "smallmoney", "money" );
-
-try
-{
-    $conn = ae_connect();
-    $conn->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_SILENT );
-
-    foreach ( $dataTypes as $dataType ) {
+try {
+    $conn = connect('', array(), PDO::ERRMODE_SILENT);
+    foreach ($dataTypes as $dataType) {
```

Comment:
Perhaps add a comment explaining why you set ERRMODE_SILENT
because not all the AE specific tests have this error mode

Reply:
The comment sounds unclear. Please rephrase when you have a chance.